cooltools/api/dotfinder.py

```diff
@@ -1402,21 +1401,18 @@ def scoring_and_extraction_step(
     # standard multiprocessing implementation
     if nproc > 1:
         logging.info(f"creating a Pool of {nproc} workers to tackle {len(tiles)} tiles")
-        pool = mp.Pool(nproc)
-        map_ = pool.imap
         map_kwargs = dict(chunksize=int(np.ceil(len(tiles) / nproc)))
     else:
         logging.info("fallback to serial implementation.")
-        map_ = map
         map_kwargs = {}
-    try:
-        # consider using
-        # https://github.com/mirnylab/cooler/blob/9e72ee202b0ac6f9d93fd2444d6f94c524962769/cooler/tools.py#L59
-        filtered_pix_chunks = map_(job, tiles, **map_kwargs)
-        significant_pixels = pd.concat(filtered_pix_chunks, ignore_index=True)
-    finally:
-        if nproc > 1:
-            pool.close()
+    
+    map_ = map
+    
+    # consider using
+    # https://github.com/mirnylab/cooler/blob/9e72ee202b0ac6f9d93fd2444d6f94c524962769/cooler/tools.py#L59
```

Comment:
is this deprecated @nvictus

Reply:
I left that comment there to try `cooler.tools.MultiplexDataPipe` in the future for describing the dots-pipeline. I think this `DataPipe` is used currently in the coverage calculations...

Unless @nvictus plans to retire `cooler.tools.MultiplexDataPipe` , I would leave that comment there - maybe change it to a more obvious: `# TODO: try using cooler.tools.MultiplexDataPipe for pipelining the steps`

Reply:
`cooler.tools  is deprecated in favor of `cooler.parallel`

Reply:
ok , than this is comment about `cooler.parallel.MultiplexDataPipe`, bit is still relevant - would be nice to replace to:
```python
# TODO: try using cooler.parallel.MultiplexDataPipe for pipelining the steps
```